src/server/system_services/schemas/account_schema.js

```diff
@@ -27,7 +27,14 @@ module.exports = {
         has_login: { type: 'boolean' },
         password: { wrapper: SensitiveString }, // bcrypted password - DEPRECATED
         next_password_change: { date: true }, // DEPRECATED
-
+        // TODO: Check NSFS behaviour for account and iam user
+        owner: { objectid: true },
+        tagging: {
```

Comment:
Maybe add the schema change related to tagging when you add the APIs of tagging?